bitrise.yml

```diff
@@ -35,57 +35,103 @@ workflows:
     - git-clone@4: {}
     - script@1:
         inputs:
-        - content: >-
-            sudo update-alternatives --set javac
-            /usr/lib/jvm/java-11-openjdk-amd64/bin/javac
-
-            sudo update-alternatives --set java
-            /usr/lib/jvm/java-11-openjdk-amd64/bin/java
-
+        - content: |-
+            sudo update-alternatives --set javac /usr/lib/jvm/java-11-openjdk-amd64/bin/javac
+            sudo update-alternatives --set java /usr/lib/jvm/java-11-openjdk-amd64/bin/java
             export JAVA_HOME='/usr/lib/jvm/java-11-openjdk-amd64'
-
-            envman add --key JAVA_HOME --value
-            '/usr/lib/jvm/java-11-openjdk-amd64'
+            envman add --key JAVA_HOME --value '/usr/lib/jvm/java-11-openjdk-amd64'
```

Comment:
I wonder why this script is needed. Seems like a fix. But I guess this was not introduced by recent changes

Reply:
I have no idea indeed, seems weird because android-sdk doesn't have anything similar but it works so I didn't want to touch anything.